source/draft_model.rst

```diff
@@ -0,0 +1,76 @@
+.. _draft_model:
+
+================
+API Object Model
+================
+
+Below is a listing of the structure of objects in the TimeSync API.
+
+Each object is listed with the fields it consists of. Type represents the type
+of data passed; it may represent data types (such as "string" or "array") or
+formats (such as "ISO date"). POST Required represents whether "empty" values
+are accepted: ``true`` means that values must be complete, while ``false`` means
+that values such as an empty string or array are valid.
+
+Note that this documentation does not reflect the way the values are stored in
+the database, only the way they are passed into and out of the API. For
+documentation on storing the objects, check the implementation.
+
+------
+
+Common
+------
+
+These fields are shared between all objects. Do not pass any of them in a POST
+request; they are provided automatically.
+
+==========  ================  =============  ======================================  ===========================================
+   Name          Type         POST Required               Description                                    Notes
+==========  ================  =============  ======================================  ===========================================
+uuid        UUID              false          A tracking ID between object revisions
+revision    positive number   false          A version number of an object
+created_at  ISO date          false          The date the object (UUID) was created
+updated_at  ISO date or null  false          The date this revision was created
+deleted_at  ISO date or null  false          The date this revision was deleted      Always null unless ?include_deleted is used
+parents     array             false          The previous revisions of this object   Only provided if ?revisions=true is used
+==========  ================  =============  ======================================  ===========================================
+
+-----
+
+Times
+-----
+
+===========  ===============   =============  ======================================  ============================================
```

Comment:
The 'Times' and 'Activities' tables still aren't being generated.

https://github.com/osuosl/timesync/blob/tristan/rewrite-add-models/source/draft_model.rst#times